script/Coprocessor.s.sol

```diff
@@ -13,16 +13,13 @@ contract MyScript is Script {
 
         // this creates the contract. it will have the same address every time if we use a 
         // new instance of anvil for every deployment.
-        // the baseURI is the prefix for the tokenURI. in this case it point to a canister deployed locally.
-        // we can infer the canister id here because we specify it in `dfx.json`. usually one would
-        // first need to create the canister, deploy the NFT contract passing the canister url as baseURI
-        // and then deploying the canister passing the NFT contract address as a deploy argument.
+
         Coprocessor coprocessor = new Coprocessor(chain_fusion_canister_address);
 
         // we can call the mint function to mint a token
         // the address we mint to belongs to the deployerPrivateKey
         // again, the transaction is signed with the deployerPrivateKey
-        for (uint256 index = 0; index < 10; index++) {
+        for (uint256 index = 0; index < 1; index++) {
```

Comment:
why not run this 10 times?

Reply:
Too many logs in the console :)

Reply:
lets find a middle ground then :D If you really only want to run it once we can get rid of the entire loop